sparkbox/apprentices-page-next
# refactor: apprentice data

## Description

- move current apprentice data into apprentices.json
- restructured json data to allow for both data sets
- moved propTypes for currentApprentice and previousApprentice groups
into exported variables to be used in the Home component
- refactored exports for CurrentApprentice to be consistent

FSA21V2-155

### Spec

See Story: [FSA21V2-155](https://sparkbox.atlassian.net/browse/FSA21V2-155)

## To Test

1. Make sure all PR Checks have passed (Github Actions, Netlify etc).
2. Pull down all related branches.
3. Confirm all tests pass: `npm run test:ci`
4. Confirm linters pass: `npm run lint`
